src/CoreWCF.Templates/src/templates/.editorconfig

```diff
@@ -0,0 +1,133 @@
+# Pour en savoir plus sur .editorconfig, consultez https://aka.ms/editorconfigdocs
```

Comment:
Is there one of these for the project as a whole - if so why is this different, and if not, should this be moved up to apply at the root of the repo?

Reply:
This file overrides is there to override the one from CoreWCF repo as i guess template users does not want the MIT license header.